filebeat/docs/fields.asciidoc

```diff
@@ -151003,6 +151003,16 @@ type: boolean
 
 --
 
+*`threatintel.misp.event_creator_email`*::
++
+--
+The email of the creator of the event.
```

Comment:
Should this be moved to `user.email` (or some `*.user.email`)? Maybe in conjunction with an addition of `user.roles: ["reporting_user"]`.

Reply:
That should be fine yeah @efd6 

After that I think we can merge.